composer.json

```diff
@@ -8,8 +8,8 @@
     ],
     "license": "BSD-3-Clause",
     "require": {
-        "cwp/cwp": "^2",
-        "silverstripe/fulltextsearch": "^3"
+        "cwp/cwp": "2.0.0-rc1",
```

Comment:
I dislike this change

Reply:
Hmm, ✔️ 
Was testing from tag, forgot to revert :/